packages/hash/frontend/src/pages/[account-slug]/entities/[entity-uuid].page/entity-editor/properties-section/property-table/cells/value-cell/array-editor/list-wrapper.tsx

```diff
@@ -0,0 +1,11 @@
+import { Box, experimental_sx as sx, styled } from "@mui/material";
+
+export const ListWrapper = styled(Box)(
+  sx({
```

Comment:
Right now this is only being used in `ArrayEditor`. If you don't plan to use it somewhere else I don't think it needs to be its own component.

Reply:
Yes, but all of the components at `/array-editor` folder is only being used in array editor. Isn't that the idea for having `my-component.tsx` and `/my-component/other-component.tsx` structure?

Reply:
I just mean that I think this wrapper could just be inside the `ArrayEditor` component, if it's not going to be used somewhere else. But happy to approve